projects/samples/howto/controllers/inverted_pendulum/inverted_pendulum.c

```diff
@@ -28,7 +28,7 @@
 #define MAX_FORCE 40
 #define CENTERING_ANGLE 0.7
 #define CENTERING_FORCE 4
-#define TIME_STEP 64
+#define TIME_STEP 32
```

Comment:
That seems to be unrelated... But maybe it's fine like that...

Reply:
Yes, a bit unrelated, but I changed this to improve the behavior of this simulation.